src/index.js

```diff
@@ -1,16 +1,6 @@
 const path = require('path')
 const parse = require('./parsers/index')
 // TODO Fix ampersand in selectors
```

Comment:
Yes I was also curious about that. @mxstbr Can we remove this? Was this related to the stylelint rules we wanted to enforce?